include/onnxruntime/core/framework/func_api.h

```diff
@@ -1,31 +1,6 @@
 #pragma once
 #include "core/common/common.h"
 namespace onnxruntime {
-//TODO: Should use the lotus cpi element type definition.
-enum DType {
```

Comment:
Hey, I have a similar change in another PR.  #989
As yours is smaller, I'd prefer to let this one go first.